modules/eks/variables.tf

```diff
@@ -43,3 +43,6 @@ variable "iam_account_id" {
   description = "Account ID of the current IAM user"
 }
 
+variable "iam_role_arns" {
```

Comment:
Can you please put the proper type here? Also maybe we can call this "iam_role_mapping" or something to indicate that it's not just ARNs.

Reply:
done